src/index.ts

```diff
@@ -119,17 +213,17 @@ function getCurrentState(config, defaultOptions) {
 }
 
 /**
- * @param  {Axios} axios
+ * @param  {Axios} axiosInstance
  * @param  {AxiosRequestConfig} config
  */
-function fixConfig(axios, config) {
-  if (axios.defaults.agent === config.agent) {
+function fixConfig(axiosInstance, config) {
```

Comment:
All of those JSDoc `@type` annotation should now be removed in favor of TS type annotations.